README.md

```diff
@@ -1,5 +1,8 @@
 # SAML Jackson (not fiction anymore)
 
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fboxyhq%2Fjackson&env=DB_ENGINE,DB_TYPE,DB_URL,JACKSON_API_KEYS,DB_ENCRYPTION_KEY&envDescription=DB%20configuration%20and%20keys%20for%20encryption%20and%20authentication.Set%20to%20''%20if%20not%20applicable.&envLink=https://boxyhq.com/docs/jackson/env-variables)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/boxyhq/jackson/tree/56-one-click-deploy)
+
```

Comment:
@niwsa Maybe you can ignore the `template` param. The docs say

_Resolve template implicitly. If you don’t specify a template parameter, Heroku infers it from the location of the button using the referer header. **This makes the button stable under forks of the repository.**_

Reply:
Nice catch. Will update the button.